openlibrary/plugins/openlibrary/js/lists/index.js

```diff
@@ -209,11 +209,7 @@ function addCreateListClickListener(button, parentDropper) {
                 seeds: [seed],
             }
 
-            const successCallback = function(listKey, listTitle) {
```

Comment:
Why was this removed?

Reply:
When the seed requirement to make a list was removed in lists.py:247 and models.py, the front-end still showed a book being added in the dropdown. I'll revert this so creating a list on the book's page still requires a seed.